Tensile/Source/CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ if(TENSILE_NEW_CLIENT)
 
     set(TENSILE_USE_HIP      ON CACHE BOOL "Use the Hip runtime.")
     set(TENSILE_USE_LLVM     ON CACHE BOOL "Use LLVM for parsing config files.")
+    if(Tensile_LIBRARY_FORMAT MATCHES "msgpack")
+        set(TENSILE_USE_MSGPACK  ON CACHE BOOL "Use message pack for parsing config files.")
+    endif()
```

Comment:
Is it possible to switch TENSILE_USE_LLVM = OFF in this case and still have things work correctly? As far as exclusivity goes, that would then allow us to use msgpack and not depend at all on llvm.

Reply:
Good suggestion. I made a couple small changes and that works.